code/game/objects/banners_vr.dm

```diff
@@ -0,0 +1,2 @@
+/obj/item/weapon/banner/solgov
+	name = "\improper SolCom banner"
```

Comment:
Doesn't this need, like, an icon state and stuff

Reply:
It is in banner.dm